.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Ask a question
+    url: https://stackoverflow.com/questions/tagged/amazon-sagemaker
```

Comment:
should we point this to the [PyTorch forums](https://discuss.pytorch.org/) instead?